src/Commands/BuildToolsBase.php

```diff
@@ -783,7 +783,7 @@ public function pushCodeToPantheon(
         $this->passthru("git -C $repositoryDir add --force -A .");
 
         // Now that everything is ready, commit the build artifacts.
-        $this->passthru("git -C $repositoryDir commit -q -m '$message'");
+        $this->passthru($this->interpolate("git -C {repositoryDir} commit -q -m [[message]]", ['repositoryDir' => $repositoryDir, 'message' => $message]));
```

Comment:
`repositoryDir` is using a different interpolate convention than `message`. It also looks like you dropped the quotes around $message.

Reply:
Right, I made the assumption that `repositoryDir` was safe so used the raw value format instead of the shell-escaped value. I think this assumption is likely fine...and if it isn't, we have a much larger task of updating the rest of Build Tools to properly escape it.

`ProcessUtils::escapeArgument()` automatically adds the quotes for `$message` -- https://github.com/symfony/process/blob/3.4/ProcessUtils.php#L78.